setup.py

```diff
@@ -153,7 +153,8 @@ def run(self):
             "pandas-profiling>=3.1",
             "ipython>=7.0",
             "markdown>=3.3",
-            "Jinja2>=3.0",
+            "Jinja2>=2.11; platform_system != 'Windows'",
+            "Jinja2>=3.0; platform_system == 'Windows'",
```

Comment:
We can't achieve compatibility with 2.11 on Windows due to Spark issues (@harupy has more info)